src/proxy-ui-api/frontend/src/views/ServiceDescriptionDetails/ServiceDescriptionDetails.vue

```diff
@@ -131,11 +148,13 @@ export default Vue.extend({
   },
   data() {
     return {
-      confirmDelete: false,
+      confirmWSDLDelete: false,
+      confirmRESTDelete: false,
       confirmEditWarning: false,
       warningInfo: [],
       touched: false,
-      serviceDesc: undefined,
+      serviceDesc: {} as any,
```

Comment:
Maybe import ServiceDescription from types.ts and use that instead of any?
serviceDesc: undefined as undefined | ServiceDescription,